src/reducetensor.cpp

```diff
@@ -639,9 +713,9 @@ void ReduceTensorDescriptor::ReduceTensor(const Handle& handle,
         param = std::string(" -std=c++14 ");
         param += " -DCK_PARAM_BLOCKSIZE=" + std::to_string(blockSize);
         param += " -DCK_PARAM_BLKGROUPSIZE=" + std::to_string(blkGroupSize);
-        param += " -DCK_PARAM_SRC_DATATYPE=" + std::to_string(detail::GetDataTypeId(srcDataType));
-        param += " -DCK_PARAM_DST_DATATYPE=" + std::to_string(detail::GetDataTypeId(dstDataType));
-        param += " -DCK_PARAM_REDUCE_COMPTYPE=" + std::to_string(detail::GetDataTypeId(compType));
+        param += " -DCK_PARAM_SRC_DATATYPE=" + std::to_string(detailS::GetDataTypeId(srcDataType));
```

Comment:
[Recommendation] `detailS` and `detailD` seem too brief. Judge yourself.

Reply:
I will change to detailStatic and detailDynamic.